token-lending/program/src/processor.rs

```diff
@@ -659,7 +675,6 @@ fn process_redeem_reserve_collateral(
     }
     let token_program_id = next_account_info(account_info_iter)?;
 
-    _refresh_reserve_interest(program_id, reserve_info, clock)?;
```

Comment:
since the lending market rate limiter is denominated in usd, we need an up-to-date price so i removed this function call